JavaShared/src/main/java/org/openzen/zenscript/javashared/JavaEnumMapper.java

```diff
@@ -0,0 +1,39 @@
+package org.openzen.zenscript.javashared;
+
+import org.openzen.zenscript.codemodel.definition.EnumDefinition;
+import org.openzen.zenscript.codemodel.member.EnumConstantMember;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class JavaEnumMapper {
+
+	private final Map<EnumDefinition, Set<EnumConstantMapping>> definitionMap;
```

Comment:
Why `Map` with a `Set` value instead of two nested maps? Judging from the `getMapping` call, having nested maps could be beneficial as you don't need to loop over the entire set.